pwiz_tools/Skyline/Model/Databinding/Entities/ColumnCaptions.resx

```diff
@@ -117,1080 +117,1117 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="AbsFoldChange">
+  <data name="AbsFoldChange" xml:space="preserve">

```

Comment:
This is a bit heavy on changes. If we need all these xml:space="preserve" added, let's do it in a separate commit.